src/styles/blocks/address.scss

```diff
@@ -0,0 +1,52 @@
+.address {
+  grid-column: 1 / 3;
+
+  @include for-tablet () {
+    grid-column: 1 / 7;
+  }
+
+  @include for-desktop() {
+    grid-column: 7 / 13;
+  }
+
+  &__title {
+    font-style: normal;
+    font-family: $secondary-font-family;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #9393a3;
+    padding-bottom: 8px;
+
+    &--phone {
+      &:hover {
+        color: navy;
+      }
+    }
+
+    &--email {
+      &:hover {
+        color: bisque;
+      }
+    }
+
+    &--address {
+      &:hover {
+        color: brown;
+      }
+    }
```

Comment:
Add hover effect only on the value of the fields, add also cursor pointer and this values (adres, number, phon) should be links which open google maps in new window and phone link should start a call. So those should be rather links instead of paragraphs